Makefile

```diff
@@ -405,9 +405,12 @@ govet:
     ./test/k8sT/... \
     ./tools/...
 
-ineffassign:
-	@$(ECHO_CHECK) ineffassign
-	$(QUIET) ineffassign .
+lint:
+	#TODO remove once Vagrant version is updated
```

Comment:
Please, address the `TODO` and remove the `curl ...` command as per [this comment](https://github.com/cilium/cilium/pull/11528/files#r447013558) this is now included in the new vagrant box version. This `curl` install command would run every time one runs `make lint` which is not a desired behavior.
LGTM otherwise :+1:

Reply:
Thanks for above comment, I was not 100% sure that the box version was bumped :100:. I will remove this _TODO_ shortly.

Reply:
Testing with the newly pulled vagrant box, seems good. Thanks again.

```
...
==> runtime1: flag to force provisioning. Provisioners marked to run always will still run.
==> runtime1: Running provisioner: build (shell)...
    runtime1: Running: inline script
    runtime1: golangci-lint cache clean
    runtime1: golangci-lint run
...
```